File: C#/0500. Keyboard Row.cs
public class Solution {
    public string[] FindWords(string[] words) {        
        List<string> rep=new List<string>();
        foreach(string word in words){
            if(AtOneLine(word)){
                rep.Add(word);
            }
        }
        return rep.ToArray();
    }
    
    public bool AtOneLine(string word){
        string str1="qwertyuiopQWERTYUIOP";
        string str2="asdfghjklASDFGHJKL";
        string str3="zxcvbnmZXCVBNM";
        if(word.Length==1){
            return true;
        }
        string target="";
        if(str1.Contains(word[0])){
            target=str1;
        }
        else if(str2.Contains(word[0])){
            target=str2;
        }
        else if(str3.Contains(word[0])){
            target=str3;
        }
        
        bool rep=true;
        for(int i=1;i<word.Length;i++){
            if(!target.Contains(word[i])){
                rep=false;
                break;
            }
        }
        return rep;
        
    }
}

File: archives/leetcode2/0500. Keyboard Row.py
class Solution(object):
    def findWords(self, words):
        """
        :type words: List[str]
        :rtype: List[str]
        """
        rep=[]
        for word in words:
            cur=[0,0,0]
            for s in word:
                if s in 'qwertyuiopQWERTYUIOP' :
                    cur[0]=1
                elif s in 'asdfghjklASDFGHJKL':
                    cur[1]=1
                else:
                    cur[2]=1
            if sum(cur)==1:
                rep.append(word)
        return rep
